requirements.txt

```diff
@@ -1,13 +1,18 @@
 falcon==1.1.0
 
+SQLAlchemy!=1.1.5,!=1.1.6,!=1.1.7,!=1.1.8,>=1.0.10 # MIT
```

Comment:
No, this sort of pattern is not required. I'll have to clean up all this stuff in a future commit.